plugins/modules/zos_blockinfile.py

```diff
@@ -540,7 +536,7 @@ def main():
         marker_begin = 'BEGIN'
     if not marker_end:
         marker_end = 'END'
-    force = '-f' if force else ''
+    force = True if force else False
```

Comment:
Can `force` be `None`? If it can be, I get this line, and it can stay. If not, could we remove it?

Reply:
By the way we use, is required to get a value

Reply:
@AndreMarcel99 didn't understand. Could you be more descriptive? I think you are referring that force does not have a default value here, hence you force assign one. But based on line 494 ` force=dict(arg_type='bool', default=False, required=False),` it gets assigned the value of False then is captured in the force vars by `force = parsed_args.get('force')`. 

If that is correct I think is removable right?

Reply:
I'm actually not sure what happens when `default` and `required` are both defined, Ansible wants us to only use one or the other, so I don't know the precedence they have. If `default` is honored, then yes, we could remove it. If not, it's just like Marcel said, we need this other line so `force` actually has a value.

Reply:
I remove because the default value take care about it, Fernando is right